libdnf-cli/argument_parser.cpp

```diff
@@ -360,6 +429,9 @@ void ArgumentParser::Command::parse(
                     cmd->parse(argv[i], argc - i, &argv[i], additional_named_args ? &extended_named_args : nullptr);
                     i = argc;
                     used = true;
+                    if (owner.complete_arg_ptr) {
```

Comment:
I'm not sure at all why this is here, so I think it deserves a comment as well.

Reply:
I use the Help class to print help for potential matches for more than one match. But I added a suggested comment.

Reply:
I added comment.